1.9/usage/debugging/cli-debugging/debug-subcommands/index.md

```diff
@@ -0,0 +1,113 @@
+---
+post_title: Debug Subcommands for Stuck Deployments
+feature_maturity: stable
+menu_order: 0
+---
+
+The DC/OS CLI provides a set of debugging subcommands to troubleshoot a stuck service or pod deployment. You can also use debug services and pods from the [DC/OS web interface](/docs/1.9/usage/debugging/gui-debugging/).
+
+# Using the debugging commands
+
+## Prerequisites
+- A DC/OS cluster.
+- The DC/OS CLI installed.
+- A service or pod that is stuck in deployment.
+
+## Sample application definitions
+If you do not currently have a service or pod that is stuck in deployment, you can use the following two [Marathon application definitions](/docs/1.9/usage/marathon/application-basics/) to test the instructions in this section.
+
+The first service creates an infinite deployment by requesting more memory than is available. The second requests too many instances.
```

Comment:
Why not just put these descriptions at each individual bullet (i.e. `This app definition creates an infinite deployment...`)